setup.py

```diff
@@ -26,12 +26,12 @@
     'intervaltree>=2.1.0,<2.2.0',
     'pyvcf<0.7.0',
     'mmh3<2.6',
+    'google-cloud-storage<=1.2.0'
```

Comment:
nit: Consider adding a comment here why you chose 1.2.0 (i.e., the discussion we had yesterday and other version conflicts, not that we, in our own code, rely on anything specific in 1.2.0).

Reply:
Done.